src/components/charts/heatMap3DPlot/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Nouislider from 'nouislider-react';
+import debounce from 'lodash/debounce';
+import MultiAxisScatter from './MultiAxisScatter';
+import 'nouislider/distribute/nouislider.css';
+import './heat-map-3d-plot.module.scss';
+
+class MultiAxisLayout extends React.PureComponent {
```

Comment:
Please rename this Class to something like LargeScaleStructureSOMETHING instead of HeatMapSOMETHING as these are not heat maps, and we may have other components in the future that are.  Ideally this name should match the name of the parent directory